tests/test_rule_replication.py

```diff
@@ -0,0 +1,44 @@
+from copy import copy
+
+from proof_of_concept.policy.replication import PolicyStore
+from proof_of_concept.policy.rules import (
+        InAssetCollection, InPartyCollection, MayAccess, ResultOfDataIn,
+        ResultOfComputeIn)
+from proof_of_concept.replication import ReplicableArchive
+from proof_of_concept.definitions.policy import Rule
+
+
+def test_rules_are_values():
```

Comment:
Maybe add a short description of what this test exactly tests, and maybe the playbook. I know I can read the code to figure it out, but I think in this case it helps to quickly grasp it.

Reply:
I guess this comment also counts for the other `test_..._are_values`

Reply:
I've added an explanation, I agree that especially this one is a bit convoluted as I threw in some background changes as well, to make sure I covered all the rule types.